src/validation.py

```diff
@@ -131,29 +134,27 @@ def test_for_negative_values(df, small: bool = False):
     for column in columns_to_test:
         negative_test = df[df[column] < 0]
         if not negative_test.empty:
-            print(" ")
-            print(
-                f"WARNING: There are {len(negative_test)} records where {column} is negative."
+            logger.warning(" ")
```

Comment:
@miloknowles could you quickly go through and remove any of these blank logger lines? This was a formatting thing to make the `print("", end="")` thing work, but is no longer necessary since we are just having the "OK" message show up on a new line instead of the same line as the "Checking...." message

Reply:
Done